### PR TITLE
Switch to Golang port as Tier 1

### PR DESCRIPTION
**Pull request by saneyuki**
It was introduced by https://github.com/nekoya/popuko/pull/24

### CODE REVIEW

**Comment by saneyuki** [comment]
@nekoya r?

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-gopuko/popuko
-gopuko/config.go
-!gopuko/config.go.example
+
+# Exec binary
+popuko
+
+# configration file
+config.go
+!config.go.example
```
Comment (nekoya) [inline .gitignore]: この行って必要なんだっけ

Comment (saneyuki) [inline .gitignore]: 要らなそう

File: Makefile [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+bootstrap:
+	rm -rf vendor/
+	go get -u github.com/mattn/gom
+	gom install
+
+new_config: $(CONFIGURE_FILE) ## Create the config file from our boilerplate.
+
+build: $(DIST_NAME) ## Build the exec binary for youe machine.
+
+build_linux_x64: ## Just an alias to build for some cloud instance.
```
Comment (nekoya) [inline Makefile]: `make help`でタスクの一覧を拾ってくるところ、grepの条件に数字が含まれていなくてこいつが出てこない。

`build_linux_x64`はpublicでいいんだよね？

Comment (saneyuki) [inline Makefile]: Yes.

File: Makefile [hunk 3]
```diff
@@ -1,16 +1,36 @@
...
 	@echo "Specify the task"
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 	@exit 1
 
-clean: ## remove virtualenv
-	rm -rf venv
+clean: ## Remove the exec binary.
+	rm -rf ./$(DIST_NAME)
+
+bootstrap:
```
Comment (saneyuki) [inline Makefile]: I'll plan to hear to @suzuken what most better approach we should do at this time. So this is temporary approach.

**Comment by saneyuki** [comment]
@popuko r=saneyuki

**Comment by popuko** [comment]
Try to merge this pull request which has been approved by `saneyuki`

**Pull request merged**